<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="vinz" failures="2" tests="2" name="C:/Users/Owner/workspace/learningautomation/src/main/java/org/learningselenium/com/Demo3.java(failed)" time="7.571" errors="0" timestamp="2021-10-27T22:57:57 MDT">
  <testcase classname="org.learning.com.Demo3" name="test2" time="2.125">
    <failure type="java.lang.AssertionError" message="expected [facebook] but found [https://www.facebook.com/]">
        <![CDATA[java.lang.AssertionError: expected [facebook] but found [https://www.facebook.com/]
at org.learning.com.Demo3.test2(Demo3.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="org.learning.com.Demo2" name="test2" time="1.653">
    <failure type="java.lang.AssertionError" message="expected [facebook] but found [https://www.facebook.com/]">
        <![CDATA[java.lang.AssertionError: expected [facebook] but found [https://www.facebook.com/]
at org.learning.com.Demo2.test2(Demo2.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="org.learning.com.Demo2" name="test" time="0.0">
    <ignored/>
  </testcase> <!-- test -->
  <testcase classname="org.learning.com.Demo3" name="zest" time="0.0">
    <ignored/>
  </testcase> <!-- zest -->
</testsuite> <!-- C:/Users/Owner/workspace/learningautomation/src/main/java/org/learningselenium/com/Demo3.java(failed) -->
